Backend/src/main/java/com/example/backend/companies/controller/CompanyController.java

```diff
@@ -107,6 +107,26 @@ public ResponseEntity<Company> editCompany(@RequestHeader String googleTokenEnco
         }
     }
 
+    @PutMapping("/softLock/{companyId}")
+    @ResponseBody
+    public ResponseEntity<Boolean> softLockCompany(@RequestHeader String googleTokenEncoded, @PathVariable Long companyId){
+        AppUser user = getUser(googleTokenEncoded);
+        try
+        {
+            if (user == null) throw new AuthenticationException();
```

Comment:
ova provjera se radi u servisu, i kad ovo prebaciš u servis dodaj poruku "You don't have access to CDB.", pa kad hvataš tu grešku u controlleru u responsu predaj tu poruku